src/scripts/postclone.js

```diff
@@ -96,6 +169,122 @@ function askPluginName() {
     }
 }
 
+function askTypeScriptDemo() {
+    if (inputParams.include_typescript_demo !== undefined) {
+        askAngularDemo();
+    } else {
+        prompt.start();
+        prompt.get({
+            name: 'include_typescript_demo',
+            description: 'Do you want to include a "TypeScript NativeScript" application linked with your plugin to make development easier (y/n)?',
+            default: 'y'
+        }, function (err, result) {
+            if (err) {
+                return console.log(err);
+            }
+
+            inputParams.include_typescript_demo = result.include_typescript_demo;
+            askAngularDemo();
+        });
+    }
+}
+
+function askAngularDemo() {
+    if (inputParams.include_angular_demo !== undefined) {
+        // askVueDemo();
+        createDemoAppsFromTemplates();
+    } else {
+        prompt.start();
+        prompt.get({
+            name: 'include_angular_demo',
+            description: 'Do you want to include a "Angular NativeScript" application linked with your plugin to make development easier (y/n)?',
+            default: 'n'
+        }, function (err, result) {
+            if (err) {
+                return console.log(err);
+            }
+
+            inputParams.include_angular_demo = result.include_angular_demo;
+            // askVueDemo();
+            createDemoAppsFromTemplates();
+        });
+    }
+}
+
+// function askVueDemo() {
+//     if (inputParams.include_vue_demo !== undefined) {
+//         createDemoAppsFromTemplates();
+//     } else {
+//         prompt.start();
+//         prompt.get({
+//             name: 'include_vue_demo',
+//             description: 'Do you want to include a "Vue NativeScript" application linked with your plugin to make development easier (y/n)?',
+//             default: 'n'
+//         }, function (err, result) {
+//             if (err) {
+//                 return console.log(err);
+//             }
+
+//             inputParams.include_vue_demo = result.include_vue_demo;
+//             createDemoAppsFromTemplates();
+//         });
+//     }
+// }
+
+function createDemoAppsFromTemplates() {
+    if (inputParams.include_typescript_demo && inputParams.include_typescript_demo.toLowerCase() === "y") {
+        appsToCreate.push({
+            command: "cd ../ && tns create " + tsAppName + " --template tns-template-blank-ts && cd " + tsAppName + " && cd ../src/",
+            successMessage: "TypeScript-NativeScript application created at: /demo",
+            type: "TypeScript"
+        });
+        appsToInstallPluginIn.push(demoTsFolder);
+    }
+
+    if (inputParams.include_angular_demo && inputParams.include_angular_demo.toLowerCase() === "y") {
+        appsToCreate.push({
+            command: "cd ../ && tns create " + angularAppName + " --template tns-template-blank-ng && cd " + angularAppName + " && cd ../src/",
+            successMessage: "Angular-NativeScript application created at: /demo-angular",
```

Comment:
successMessage: "TypeScript-NativeScript application created at: " + demoAngularFolder,

Reply:
Yes, good suggestion I will make the change.